demos/asos/train.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Please add a comment with a disclaimer that this code is taken from Timo's repo and we are only responsible for adding the tracking.